core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -797,16 +801,20 @@ private[spark] class TaskSetManager(
     maybeFinishTaskSet()
   }
 
-  private[scheduler] def markPartitionCompleted(partitionId: Int, taskInfo: TaskInfo): Unit = {
+  private[scheduler] def markPartitionCompleted(partitionId: Int, taskInfo: Option[TaskInfo])
+  : Unit = {
```

Comment:
super nit: I'm pretty sure that even if just the return type goes over the line, our style is to switch to multi-line definition